rad/prelude/cmd-parsing.rad

```diff
@@ -0,0 +1,69 @@
+{:module 'prelude/cmd-parsing
+ :doc "Functions for parsing commands."
+ :exports
+ '[parse-failure /cmd-0 /cmd-1 /cmd-2 /cmd-2-opts]}
+
+(load! "rad/prelude.rad")
+(import prelude/validation :as 'validation)
+
+(def parse-failure
+ (fn [error help]
+   (put-str! error)
+   (put-str! help)
+   (exit! 1)
+ ))
+
+ (def missing-arg-error
+   (fn [arg cmd]
+     (string-append "Missing argument \"" (show arg) "\" for command \"" cmd "\"")))
```

Comment:
Is this and the next function indented too much?

Reply:
Fixed the indentations.